CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this extension will be documented in this file.
 
 ## Unreleased
 
+## 0.20.1
+
 ### Fixed
```

Comment:
Can we add a line about https://github.com/confluentinc/vscode/pull/479 please?

Reply:
I'll suggest: "Fixed bug where switching between consume tabs would reset consume mode to default"

Reply:
Ah I just saw that suggestion - does https://github.com/confluentinc/vscode/pull/481/commits/d9387e133850655515244b962d6d66e5718a3f48 read okay?

Reply:
Yeah, that's good, TYVM!